common/djangoapps/cme_registration/tests/registration_tests.py

```diff
@@ -405,8 +404,7 @@ def test_db_records_with_others_created(self):
         self.post_vars['specialty_free'] = 'Patient care'
         self.post_vars['sub_specialty'] = 'Other'
         self.post_vars['sub_specialty_free'] = 'Legs and feet'
-        self.post_vars['affiliation'] = 'Other'
-        self.post_vars['other_affiliation'] = 'other_affiliation'
+        self.post_vars['affiliation'] = 'Not affiliated with Stanford Medicine'
```

Comment:
Discussed with @stvstnfrd and resolved; with the departure of the "other" options this is indeed a superficial test, but it provides some value in making sure the new "Not affiliated" affiliation doesn't cause issues with acct. creation